### PR TITLE
desktop version

### PR DESCRIPTION
**Pull request by PedroH-Neves**
- Add all desktop content 
- Use of CSS media queries
- Add new assets for desktop sizes
- Make nav menu functional

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -16,14 +16,19 @@
...
+  <header class="header">
     <a href="#" class="logo primary-color no-underscore">P</a>
     <nav class="menu-icon">
-      <img src="./assets/Icons/Burger.svg" alt="menu icon">
+      <img class="hamburger-icon" src="./assets/Icons/Burger.svg" alt="menu icon">
+      <ul class="menu-links">
+        <li class="menu-links-itens"><a class="no-underscore primary-color" href="#portfolio">Portfolio</a></li>
+        <li class="menu-links-itens"><a class="no-underscore primary-color" href="#about">About</a></li>
+        <li class="menu-links-itens"><a class="no-underscore primary-color" href="#contact">Contact</a></li>
+      </ul>
```
Comment (ha-manel) [inline index.html:28]: - [x] Kindly add the missing icon.

## Your design:
image

## Figma:
image

File: index.html [hunk 2]
```diff
@@ -56,31 +61,36 @@ <h3 id="h3" class="primary-color">
...
+          </p>
+          <ul class="project-list-tags">
+            <li class="project-list-item"><span class="tag">rust</span></li>
+            <li class="project-list-item"><span class="tag">elixir</span></li>
+            <li class="project-list-item"><span class="tag">c++</span></li>
+            <li class="project-list-item"><span class="tag">lua</span></li>
+          </ul>
+          <a href="#" class="project-link primary-color no-underscore button-hover button-pressed button-disabled">See
+            Project</a>
+        </div>
```
Comment (ha-manel) [inline index.html]: - [x] Kindly edit this section's size to make it match the Figma design 💯 

## Your design:
image

## Figma:
image

File: index.html [hunk 3]
```diff
@@ -56,31 +61,36 @@ <h3 id="h3" class="primary-color">
...
       </div>
-      <section class="projects-section-two">
+      <section id="portfolio" class="projects-section-two">
```
Comment (ha-manel) [inline index.html:94]: - [ ] Kindly edit the desktop version of the cards to make it match Figma; when the project is hovered on with the mouse the description and the dark opacity disappear and a `See Project` button appears at the bottom of the image and takes the whole image width. 

## Default state:
image

## On hover:
image

File: index.html [hunk 4]
```diff
@@ -185,7 +195,7 @@ <h3 class="project-title-two">
         </div>
       </section>
     </section>
-    <section class="about-section">
+    <section id="about" class="about-section">
       <div class="about-box">
         <h3 class="about-title secundary-color">About Me</h3>
         <p class="about-text primary-color">Hello I’m a software developer! I can help you build a product , feature or
```
Comment (ha-manel) [inline index.html:202]: - [ ] Kindly edit the font for this section, as well as the `see project` button for the cards, the `skills`, and the `form`.

## Your design:
image
image
image
image

## Figma:
image
image
image
image

File: style.css [hunk 5]
```diff
@@ -345,6 +362,201 @@
...
+  .project-link {
+    margin-top: 20px;
+  }
+
+  /* PROJECT SECTION TWO */
+
+  .projects-section-two {
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr;
+    grid-template-rows: 1fr 1fr 1fr;
```
Comment (ha-manel) [inline style.css]: - [x] Kindly fix this error, there are 2 rows in the grid but you made 3 which caused the section to have a huge empty gap at the bottom, you can remove the row and add `margin-bottom` instead 💯 

image

File: style.css [hunk 6]
```diff
@@ -345,6 +362,201 @@
...
+    margin-left: 0;
+  }
+
+  .project-content {
+    width: 50%;
+    display: flex;
+    justify-content: space-evenly;
+    flex-direction: column;
+    align-self: flex-start;
+  }
```
Comment (ha-manel) [inline style.css]: - [x] Kindly fix the issues in mobile version 👍 

image

**Review by ha-manel — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES ♻️

## Hi @PedroH-Neves 👋 
Great job so far, There are some changes that need to be implemented in order to proceed to the next project 👍  



## Highlights:  
* All tests passed ✔️
* Meaningful commit messages ✔️ 
* Professional README file ✔️

## Required Changes:
_Please check the comments below_

------
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by codecaiine — APPROVED** [review]
Hi @PedroH-Neves , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights


- Good requested changes application :heavy_check_mark: 
- No linters errors :heavy_check_mark: 
- Great design :heavy_check_mark: 

## Optional suggestions
- [ ] [OPTIONAL] I would like to recommend you work on your mobile version and your desktop version navbar contact icon position cause it is important to follow Figma design as much as possible :+1:  
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me (@codecaiine) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**